scanner/Makefile

```diff
@@ -69,14 +73,19 @@ tag:
 .PHONY: images
 images: scanner-image db-image
 
-.PHONY: scanner-go-build
+.PHONY: scanner-go-build scanner-go-check-ver
+scanner-go-check-ver:
+ifneq ($(GO_VERSION),$(EXPECTED_GO_VERSION))
```

Comment:
Does this not need to be indented?

Reply:
No.

Reply:
(If it were, it would be interpreted as the recipe rules, right?)